Aptoide/aptoide-client-v8
# [ASV-1153] downloads metrics

**What does this PR do?**

 Adds the missing install complete event to facebook and flurry. Also adds the error event due to root that was missing since april 4th.
 
**Database changed?**

No

**Where should the reviewer start?**

-InstallAnalytics.java

**How should this be manually tested?**

Do a download in a non-editorial app (app view and apps) and check if the application install was sent to facebook and flurry and if the INSTALL event was sent to the BI.
 
**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-1153

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
